gamedata/loc_US/loc/US/strings_db.lua

```diff
@@ -2300,13 +2300,15 @@ lobui_0119="All terrain and units visible"
 lobui_0120="Victory Condition"
 lobui_0121="Determines how a victory can be achieved"
 lobui_0122="Assassination"
-lobui_0123="Game ends when commander is destroyed"
+lobui_0123="Player is defeated when the Commander is destroyed"

```

Comment:
My opinion is that each of these strings should begin with "A player" rather than "Player". Currently, combined with the use of "the Commander" as a singular proper noun, this sentence doesn't sound as much like it applies individually to each player.

Reply:
I'll change all the VC descriptions once the other proposed changes are discussed & agreed upon.